nibabel/externals/netcdf.py

```diff
@@ -58,24 +58,25 @@
 
 
 TYPEMAP = {NC_BYTE: ('b', 1),
-            NC_CHAR: ('c', 1),
```

Comment:
Would you mind removing the changes to `netcdf.py`?   The reason being, that this file is a very slightly edited copy of scipy's version of this file, so I would like to keep the changes minimal relative to the original, when keeping the files up to date.